configure

```diff
@@ -1713,9 +1713,11 @@ Optional Features:
                           no)
   --enable-optimizations  enable expensive, stable optimizations (PGO, etc.)
                           (default is no)
-  --enable-loadable-sqlite-extensions
-                          support loadable extensions in _sqlite module, see
-                          Doc/library/sqlite3.rst (default is no)
+
+    --enable-loadable-sqlite-extensions
+                          support loadable extensions in the sqlite3 module,
+                          see Doc/library/sqlite3.rst (default is no)
+
```

Comment:
Judging by indentation of the rest of the section and logical affinity of `--enable-loadable-sqlite-extensions` to preceeding `--enable-optimizations`, I believe these linese were changed by mistake.

```suggestion
  --enable-loadable-sqlite-extensions
                          support loadable extensions in _sqlite module, see
                          Doc/library/sqlite3.rst (default is no)
```

Reply:
Good catch, but the we do not edit `configure`: `configure` is autogenerated from `configure.ac` by `autoconf`. (Actually, it is generated from a specific patched version residing in a container Christian set up; you'll find the incantation in the CI).

FYI: The indentation changed because the indentation and placement of the first parameter to `AS_HELP_STRING` changed. M4 lets most whitespace through as it is, unless you explicitly exclude it or filter it.

Reply:
Correction: We use the stock version from Alpine 3.13 without additional patches.

Reply:
Fixed in 948a296fade59f3fbc02866c2528c740c5995eb7

Reply:
Ah, I was under the impression that it was patched; my bad, sorry 'bout that :)

Reply:
You are right and wrong at the same time. Alpine 3.13 applies to patches to autoconf. One of the patches is a backport of runstatedir.

Reply:
BTW, I also note that because of the brackets we now of course include the preceding and succeeding newline. The easiest fix is to remove the brackets.